src/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/RuntimeInformation.cs

```diff
@@ -22,7 +23,7 @@ public static string FrameworkDescription
         {
             get
             {
-                return s_frameworkDescription ?? (s_frameworkDescription = $"{FrameworkName} {typeof(object).GetTypeInfo().Assembly.GetName().Version}");
+                return s_frameworkDescription ?? (s_frameworkDescription = $"{FrameworkName} {FileVersionInfo.GetVersionInfo(typeof(object).GetTypeInfo().Assembly.Location).FileVersion}");
```

Comment:
I don't think this actually works.  The reflection API you're using isn't available on many of the platforms you're targeting that include this file.  I suspect the use of this in a formattable string is turning that into a runtime failure rather than build time.